packages/accounts/src/fetch-account.ts

```diff
@@ -13,6 +13,11 @@ import type { GetAccountInfoApi, GetMultipleAccountsApi } from './rpc-api';
  */
 export type FetchAccountConfig = {
     abortSignal?: AbortSignal;
+    /**
+     * Fetch the details of the account as of the highest slot that has reached this level of
+     * commitment.
+     * @defaultValue "finalized"
```

Comment:
This is a tricky one because, even though the RPC default might be `finalized`, you're likely going to end up using `confirmed` because `createSolanaRpcApi` sets it as a default commitment.

Reply:
Agreed. I'm not sure what the best way to document these is TBH.

Reply:
Nice catch. On the one hand, this is supposed to be the _spec_ documentation, before the `@solana/kit` wrappers, but on the other hand these docs propagate all the way up whether we like it or not.

It's possible that we could override these arguments in each place they appear, in Kit itself, so as to inject different docs here, but it's probably more sane just to mention both behaviours here.